packages/donor/src/components/Wizard/WizardScreen.tsx

```diff
@@ -1,40 +1,103 @@
-import Stepper from "../basic/Stepper";
-import Button, { ButtonVariant } from "../basic/Button";
+import Button from "../basic/Button/ButtonV2";
+import Text from "../basic/Text";
+import styles from "./WizardScreen.module.scss";
+import { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import MobileStepper from "@material-ui/core/MobileStepper";
+import { BrowserRouter, Link } from "react-router-dom";
+import { PropTypes } from "@material-ui/core";
+import { ButtonVariant } from "../basic/Button";
 
-export interface WizardScreenProps {
-  imageUrl: string;
-  title: string;
-  content: string;
+const useStyles = makeStyles({
+  root: {
+    flexDirection: "column",
+    fontFamily: "Rubik",
+    backgroundColor: "#fff",
+  },
 
-  // Stepper props:
-  currentStep: number;
-  totalNumberOfSteps: number;
+  dot: {
+    margin: "8px",
+  },
 
-  // Action button
+  dots: {
+    margin: "30px 0",
+  },
+
+  dotActive: {
+    backgroundColor: "#A2D45E",
+  },
+});
+
+interface page {
+  imageUrl: string;
+  title: string;
+  content: { text: string; bold: boolean }[];
   buttonText: string;
   buttonVariant: ButtonVariant;
-
-  onNext?: () => void;
-  onPrev?: () => void;
-  onClick: () => void;
+  buttonColor: PropTypes.Color;
+}
+export interface WizardScreenProps {
+  pages: Array<page>;
+  currentStep: number;
```

Comment:
Why is this an external prop?

Should we handle the count inside the component?

Reply:
yeah i think so too, that was there before i thought maybe there a reason for it.